src/scss/_mixins.scss

```diff
@@ -5,7 +5,7 @@
 ///     }
 @mixin oEditorialTypographyBody() {
 	@include oTypographySerif($scale: (default: 1, XL: 2), $line-height: 1.6);
-	color: oColorsGetColorFor('body', 'text');
```

Comment:
How did this work?! it never directly depended on o-colors! 😱

Reply:
By the magic of _indirectly_ depending on o-colors!  Let's get ourselves and the ft at large to use the new module system